homeassistant/helpers/config_validation.py

```diff
@@ -955,6 +955,11 @@ def custom_serializer(schema: Any) -> Any:
     """Serialize additional types for voluptuous_serialize."""
     from . import selector  # pylint: disable=import-outside-toplevel
 
+    # Is it vol.Maybe?
```

Comment:
The other day I tried to serialize a vol.Exclusive and it wouldn't work.  I wanted a simple "select this section or that section" of the config.

I know this is not related to that, but I just wanted to express my appreciation for what you are doing here.

Approved.